source/clear-linux/reference/collaboration/community/conduct.rst

```diff
@@ -0,0 +1,98 @@
+.. _conduct:
+
+Code of conduct
+###############
+
+The |CLOSIA| community is made up of a mixture of professionals and
+volunteers from all over the world. We strongly believe diversity is our
+strength and want to promote an inclusive and safe environment.
+
+To that end, we have a few ground rules for engaging in the community. The
+following code of conduct applies equally to all community participants,
+including project team leads, core contributors, mentors, users,
+participants, and those seeking help and guidance.
+
+This code of conduct also applies to all spaces the project manages or
+authorizes, including IRC channels, the mailing lists, issue trackers,
+sponsored events, and any other forums the community uses and the project
+manages. In addition, violations of this code of conduct outside of these
+venues may affect a person's ability to participate within them after
+appropriate investigation.
+
+**Be friendly, patient and welcoming.** We strive to be a community that
```

Comment:
Do we want to utilize the serial comma here?

Reply:
Yes, added the Oxford comma.